### PR TITLE
LRQA-70406, LRQA-70409, LRQA-70410 | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/LRQA-70406
https://issues.liferay.com/browse/LRQA-70409
https://issues.liferay.com/browse/LRQA-70410

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase [hunk 1]
```diff
@@ -13,8 +13,9 @@ definition {
 		SearchAdministration.executeReindex();
 	}
 
-	@description = "This is a use case for LRQA-61554."
+	@description = "This is a use case for LRQA-61554. Login to instance and check if OAuth2 works after upgrade from 7.1.3"
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase:16]: Please use the proper names in description -» "OAuth 2"

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
See my review comments above.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase [hunk 2]
```diff
@@ -49,7 +47,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-61554."
+	@description = "This is a use case for LRQA-61554. Login to instance and check if OAuth2 works after upgrade from 7.2.0"
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase:50]: Change OAuth2 to OAuth 2 here too

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase [hunk 3]
```diff
@@ -49,7 +47,7 @@ definition {
...
 	@priority = "5"
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase:51]: I think that you can flag all tests with @uitest as well, as they are very similar to the first one

**Review by weller0246 — CHANGES_REQUESTED** [review]
Please change OAuth2 to OAuth 2 as Gabor described above and resolve my comment, thanks

**Review by ferenconodiD — APPROVED** [review]
Resolve the comments of Gábor and Marcell, other then that LGTM!

**Comment by Kvale1733** [comment]
@gaborlovasdotcom OAuth2 has been changed to OAuth 2 and all tests have been flagged with @uitest, ready for review

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:security:nocompile

**Comment by Kvale1733** [comment]
@gaborlovasdotcom I have fixed the OAuth 2 name, added the @UI test flags, and reverted the use of the openOAuth2Admin macro for the instance tests as it was causing failure in CI here. All tests are passing now.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase (hunk 3 above)
Comment (JoyceWang08211) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase:54]: Here is how to use @uitest label rule:
Make sure that at least one Poshi test is still testing basic UI flows. Mark these so we don't convert them as we refactor the rest of our tests
Use @uitest as an annotation for this

So I think mark one instance level test and one system level test as uitest is enough.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase [hunk 4]
```diff
@@ -241,17 +239,15 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-61554."
+	@description = "This is a use case for LRQA-61554. Verify OAuth 2 works at the system level after upgrade from 7.2.10.1"
 	@priority = "5"
+	@uitest
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase:244]: Here is how to use @uitest label rule:
Make sure that at least one Poshi test is still testing basic UI flows. Mark these so we don't convert them as we refactor the rest of our tests
Use @uitest as an annotation for this

So I think mark one instance level test and one system level test as uitest is enough.

I think we might need a discussion about how to use this @uitest label.
@gaborlovasdotcom

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase:244]: Or not use UI test at all, if we don't benefit from this labeling.

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
@gaborlovasdotcom I have made changes to the UI flags

**Comment by Kvale1733** [comment]
ci:test:security:nocompile

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**